src/styles/blocks/_contacts.scss

```diff
@@ -0,0 +1,101 @@
+.contacts {
+  padding: 80px 0 120px;
+
+  @include on-tablet {
+    padding: 120px 0 102px;
+  }
+
+  @include on-desktop {
+    padding: 180px 0 164px;
+  }
+
+  &__content {
+    @include page-grid;
+  }
+
+  &__title {
+    margin-bottom: 48px;
+    text-align: center;
+  }
+
+  &__form {
+    width: 100%;
+    border-radius: 40px;
+    grid-column: 1/-1;
+
+    @include on-tablet {
+      grid-column: 1/4;
+    }
+
+    @include on-desktop {
+      grid-column: 1/6;
+    }
+  }
+
+  &__label {
+    width: 100%;
+  }
+
+  &__input-name {
+    width: 100%;
+    height: 48px;
+    padding: 14px 0 14px 16px;
+    background: $color-primary-input;
+    margin-bottom: 16px;
+    border-radius: 40px;
+    color: $color-secondary-input;
+  }
+
+  &__input-email {
+    width: 100%;
+    height: 48px;
+    padding: 14px 0 14px 16px;
+    background: $color-primary-input;
+    margin-bottom: 16px;
+    border-radius: 40px;
+    color: $color-secondary-input;
+  }
+
+  &__input--transform {
+    &:hover {
+      border: 1px solid $color-input-hover;
+      color: $color-input-hover;
+    }
+
+    &:focus {
+      border: 1px solid $color-input-active;
+      color: $color-input-active;
+    }
+  }
+
+  &__textarea {
+    width: 100%;
+    min-width: 288px;
+    max-width: 405px;
```

Comment:
Setting different max widths for different form fields (textarea, inputs) causes them to have different sizes on some window widths:
![form375](https://github.com/mate-academy/layout_landing-page/assets/6094468/8d7a9162-0a1f-40e3-8607-5db9876e5b89)
I'd recommend setting `max-width: 100%;` for all form fields at least for small screens, in order to avoid layout issues.